File: Java/Matrix or 2D Array  - print using for-each loop.java
class matrixPrint_forEachLoop {
    public static void main(String[] args){
        // (2row X 3columns)
        int[][] A = {
            { 1, 2, 3 },
            { 4, 5, 6 },
            { 7, 8, 9 }
        };
        System.out.println("Matrix: A");
        for (int[] row:A) {
            for (int col:row) {
                System.out.print("\t" + col);
            }
            System.out.println();
        }
    }
}

File: C++/Array/Matrix or 2D Array - print using for-each loop.cpp
#include <iostream>
using namespace std;

int main(){
    int A[3][3] = {
        { 1, 2, 3 },
        { 4, 5, 6 },
        { 7, 8, 9 }
    };
    
    cout << "Matrix: A" << endl;
    for (auto &row:A) {
        for (int &col:row) {
            cout << "\t" << col;
        }
        cout << endl;
    }
}